src/js/prediction.js

```diff
@@ -181,12 +171,7 @@ class Click {
         after_response(info);
       };
 
-      // DRAG START
-      pickerCircle.addEventListener('mousedown', mousedown);
-      pickerCircle.addEventListener('mousemove.drag', mousemove);
-      document.addEventListener('mousemove.drag', mousemove);
-
-      // ENABLE STARTING THE DRAG IN THE BLACK CIRCLE
+      // ENABLE STARTING THE CLICK IN THE BLACK CIRCLE
       circle.addEventListener('mousedown', function (event) {
         if (event.target == this) mousedown(event);
```

Comment:
Why not this?
```python
circle.addEventListener('mousedown', mousedown)
```

Reply:
Nevermind, I figured it out. This prevents the event from occurring in the inner circle or outside the main circle.